code/modules/clothing/suits/labcoat.dm

```diff
@@ -85,3 +86,10 @@
 	base_icon_state = "labcoat_onc"
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 60)
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
+
+/obj/item/clothing/suit/storage/labcoat/rd
+	name = "Research director's Labcoat"
```

Comment:
Needs either \improper or the caps removed. The later would be better, same for everything else in this file, thanks oldcoders.